ci/docker/run/Dockerfile

```diff
@@ -39,7 +45,7 @@ RUN git clone "${REPO}" verilator && \
     git checkout "${SOURCE_COMMIT}" && \
     autoconf && \
     ./configure && \
-    make -j "$(nproc)" && \
+    make && \
```

Comment:
Was this changed as part of your flakyness resource debugging, or does need to be removed for another reason?

Reply:
Yes. See https://github.com/actions/runner-images/issues/6680